### PR TITLE
Cleanup debug logging

### PR DESCRIPTION
**Pull request by zneix**
Pull request checklist:

- [x] `CHANGELOG.md` was updated, if applicable

# Description

- Removed "Drag segment:" log, used to be emitted every time you switched to an irc split
- Removed verbose websocketpp debug output for received frame headers

### CODE REVIEW

File: src/debug/Benchmark.cpp [hunk 1]
```diff
@@ -12,12 +12,12 @@ BenchmarkGuard::BenchmarkGuard(const QString &_name)
 BenchmarkGuard::~BenchmarkGuard()
 {
     qCDebug(chatterinoBenchmark)
-        << this->name_ << float(timer_.nsecsElapsed()) / 1000000.0f << "ms";
+        << this->name_ << float(timer_.nsecsElapsed()) / 1000.0f << "ms";
```
Comment (pajlada) [inline src/debug/Benchmark.cpp:15]: This seems incorrect to me, 1 millisecond is 1000000 nanoseconds, not 1000

File: src/debug/Benchmark.cpp [hunk 2]
```diff
@@ -12,12 +12,12 @@ BenchmarkGuard::BenchmarkGuard(const QString &_name)
...
 }
 
 qreal BenchmarkGuard::getElapsedMs()
 {
-    return qreal(timer_.nsecsElapsed()) / 1000000.0;
+    return qreal(timer_.nsecsElapsed()) / 1000.0;
```
Comment (pajlada) [inline src/debug/Benchmark.cpp:20]: Same as the previous change in `BenchmarkGuard`

**Review by pajlada — CHANGES_REQUESTED** [review]

File: src/debug/Benchmark.cpp (hunk 1 above)
Comment (pajlada) [reply src/debug/Benchmark.cpp:15]: Without your change:
image
image

So I imagine your change would be incorrect, feel free to test it yourself too

Comment (apa420) [reply src/debug/Benchmark.cpp:15]: I've also previously mixed these up but it's milli (10^3), micro (10^6) and then nano (10^9).

Comment (zneix) [reply src/debug/Benchmark.cpp:15]: I must've gotten confused with microseconds. Sorry for this silly change.

**Review by pajlada — APPROVED** [review]

**Pull request merged**